src/templates/sermons.factory.js

```diff
@@ -1,14 +1,20 @@
-const path = require(`path`),
+const _ = require(`lodash`),
+    path = require(`path`),
     constants = require(`../constants.js`)
 
-exports.createPages = async (graphql, createPage) => {
-    await Promise.all(Object.keys(constants.languages).map(async (key) => {
-        const lang = constants.languages[key]
+const createPagesByLang = async (
+    langKey,
+    graphql,
+    createPage,
+    page = 1,
+    cursor = null
+) => {
+    const lang = constants.languages[langKey]
 
-        const result = await graphql(`
+    const result = await graphql(`
 {
     avorg {
-        sermons(language:${key}) {
+        sermons(language:${langKey}, after:"${cursor}") {
```

Comment:
Rather than using string interpolation a better practice is to use query variables: https://www.gatsbyjs.org/docs/graphql-reference/#query-variables.

Reply:
Reference: https://github.com/thalseth/kystleik/blob/32691a7e8b5700deb57f15985291d8e47e5a67e5/web/gatsby-node.js#L73-L91